content/chronograf/v1.9/guides/configuring-alert-endpoints.md

```diff
@@ -167,224 +155,221 @@ See the [AlertNode (Kapacitor TICKscript node) - OpsGenie v2](/{{< latest "kapac
-  * **Configuration Enabled**: Check to enable this configuration.
+    - **Routing Key**: GUID of your PagerDuty Events API V2 integration, listed as "Integration Key" on the Events API V2 integration's detail page. See [Create a new service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) in the PagerDuty documentation details on getting an "Integration Key" (`routing_key`).
+    - **PagerDuty URL**: URL used to POST a JSON body representing the event. This value should not be changed. Valid value is `https://events.pagerduty.com/v2/enqueue`.
+    - **Configuration Enabled**: Check to enable this configuration.
 
 3. Click **Save Changes** to save the configuration settings.
 4. Click **Send Test Alert** to verify the configuration.
 
-See the [PagerDuty Events API V2 Overview](https://v2.developer.pagerduty.com/docs/events-api-v2) for details on the PagerDuty Events API and recognized event types (`trigger`, `acknowledge`, and `resolve`).
+See the [PagerDuty Events API V2 Overview](https://v2.developer.pagerduty.com/docs/events-api-v2)
+for details on the PagerDuty Events API and recognized event types (`trigger`, `acknowledge`, and `resolve`).
 
-See [AlertNode (Kapacitor TICKscript node) - PagerDuty v2](/{{< latest "kapacitor" >}}/nodes/alert_node/#pagerduty-v2) in the Kapacitor documentation for details about enabling a new "Generic API" service using TICKscripts.
+See [AlertNode (Kapacitor TICKscript node) - PagerDuty v2](/{{< latest "kapacitor" >}}/nodes/alert_node/#pagerduty-v2)
+in the Kapacitor documentation for details about enabling a new "Generic API" service using TICKscripts.
 
 ### Pushover
 
-Configure Chronograf to send Pushover event handler alerts.
+**To configure a Pushover alert endpoint:**
 
-#### User Key
+1. In the **Configure Alert Endpoints** of the **Configure Kapacitor Connection** page,
+   click the **Pushover** tab.
+2. Enter the following:
 
-Enter your Pushover USER_TOKEN.
+    - **User Key**: Pushover USER_TOKEN.
+    - **Token**: Pushover API token.
+    - **Pushover URL**: Pushover API URL.
+      Default is `https://api.pushover.net/1/messages.json`.
+    - **Configuration Enabled**: Check to enable configuration.
 
-#### Token
+3. Click **Save Changes** to save the configuration settings.
+4. Click **Send Test Alert** to verify the configuration.
 
-Enter your Pushover API token.
+### Sensu
 
-#### Pushover URL
+**To configure a Sensu alert endpoint:**
 
-The URL for the Pushover API. The default value is `https://api.pushover.net/1/messages.json`.
+1. In the **Configure Alert Endpoints** of the **Configure Kapacitor Connection** page,
+   click the **Sensu** tab.
+2. Enter the following:
 
-#### Configuration Enabled
+    - **Source**: Event source. Default is `Kapacitor`.
+    - **Address**: Base URL of [Sensu API](https://docs.sensu.io/sensu-go/latest/migrate/#architecture).
```

Comment:
To be sure, is the base URL what Sensu refers to as the "sensu-backend" or "etcd datastore"?